### DISCUSSION TITLE
Problem with postcss.config.js file in Parcel+TailwindCSS project

### DISCUSSION THREAD

**Discussion by cichy380** [parcel-bundler/parcel#7860] [Q&A]
This is my first time with Parcel and first time with [TailwindCSS](https://tailwindcss.com/). I made installation according to the instructions from: https://tailwindcss.com/docs/guides/parcel

My initial project is available on https://github.com/cichy380/ania/tree/tailbug

When I run `npx parcel src/index.html` I got WARNING about `@parcel/transformer-postcss` and `postcss.config.js`.

Preview on my terminal:
```
$ npx parcel src/index.html

Server running at http://localhost:1234
Building...

@parcel/transformer-postcss: WARNING: Using a JavaScript PostCSS config file means losing out on caching features of
Parcel. Use a .postcssrc(.json) file whenever possible.


@parcel/transformer-postcss: Parcel includes CSS transpilation and vendor prefixing by default. PostCSS config
postcss.config.js contains the following redundant plugins: autoprefixer. Removing these may improve build performance.

  C:\Projekty\poc\tailwind\parcel\postcss.config.js:1:1
  > 1 | module.exports = {
  >   | ^
    2 |   plugins: {
    3 |     tailwindcss: {},

  ℹ Remove the above plugins from postcss.config.js
  ℹ Learn more: https://parceljs.org/languages/css/#default-plugins

√ Built in 978ms
```

What I forgot about? Or what am I doing wrong?

**Comment by devongovett** [comment]
You don't need autoprefixer anymore as of 2.4.0. See https://parceljs.org/languages/css/#default-plugins. I'll see about sending a pr to update tailwind's docs too.

**Reply by devongovett** [reply]
https://github.com/tailwindlabs/tailwindcss.com/pull/1227
